pkg/helm/index_test.go

```diff
@@ -79,3 +82,11 @@ func Test_newChart(t *testing.T) {
 	assert.Equal(t, c.Repo, r, "repo set")
 	assert.Equal(t, c.ID, "test/wordpress", "id set")
 }
+
+func Test_loadRepoWithEmptyCharts(t *testing.T) {
```

Comment:
Just wondering if adding the extra yaml file really worths it or we can just go with appending the empty chart (as in https://github.com/kubeapps/kubeapps/pull/3152#pullrequestreview-706194220) in an already valid yaml file.

Reply:
It looked more clean to me but if you prefer I could rewrite the test using the same file, as you suggest.
I'm not such a good golang programmer so advices are welcome :)